addon/bootstrap.js

```diff
@@ -112,7 +111,6 @@ this.Bootstrap = {
   initStudyUtils(id, version) {
     // validate study config
     studyUtils.setup({...config, addon: { id, version }});
-    // TODO bdanforth: patch studyUtils to setLoggingLevel as part of setup method
```

Comment:
In the [plans](https://github.com/mozilla/shield-studies-addon-utils/issues/101#issuecomment-367409290) for the next rev of StudyUtils.